src/coreclr/jit/emitriscv64.cpp

```diff
@@ -1258,7 +1258,9 @@ void emitter::emitLoadImmediate(emitAttr size, regNumber reg, ssize_t imm)
     INT32 high19 = ((int32_t)(high31 + 0x800)) >> 12;
 
     emitIns_R_I(INS_lui, size, reg, high19);
-    emitIns_R_R_I(INS_addiw, size, reg, reg, high31 & 0xFFF);
+    if (high31 & 0xFFF) {
```

Comment:
Shouldn't the opening brace be on a new line?

Reply:
Thank you. Fixed.